BorrowBooks/features/borrowed-books/borrowed-books.php

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Borrowed Books</title>
+        <link rel="stylesheet" href="borrowed-books.css" />
+	    <script type="module" src="borrowed-books.js"></script>
+    </head>
+    <body>
+        <section class="borrowed-books-page">
+            <nav class="sidebar">
+                <ul>
+                    <li><a href="../home/home.php">Home</a></li>
+                    <li><a href="../borrowed-books/borrowed-books.php">Borrowed Books</a></li>
+                    <li><a href="#favourite-quotes.php">Favourite Quotes</a></li>
+                    <li><a href="#statistics.php">Statistics</a></li>
+                    <li><a href="#literary-reading.php">Literary Reading</a></li>
+                    <li><a href="#logout.php">Logout</a></li>
+                </ul>
+                <section class="import-container">
+                    <button class="import-button">
+                        <img src="../../images/plus-icon.png" alt="Import">
+                    </button>
+                    <span class="import-text">Import Books</span>
+                </section>
+            </nav>
```

Comment:
This can be export in new file just for sidebar and later be included in each page. By doing this, we will make code reusable and there will be no need for each page to include this code as well as css and js for it.

Reply:
Done.